src/components/Loader/Loader.stories.mdx

```diff
@@ -1,8 +1,9 @@
 import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
 
 import Loader from './Loader';
+import {loaderTypes} from './Loader.utils';
 
-<Meta title="Design System/Loader" component={Loader} />
+<Meta title="Design System/Loaders" component={Loader} />
```

Comment:
why plural? In that case Button could be Buttons

Reply:
I thought of this like a group of loaders. Like having different geometric shapes.

Geometric Shapes (plural) => Square, triangle, circle etc...
Loaders => Spinner, dots, indeterminate etc... 

but i can revert it to `Loader` if this is blocking

Reply:
no its not blocking it was just confusing as it refers to a single point system like buttons